### ISSUE TITLE
BB presetable pedal LCD

### ISSUE THREAD

**Issue by vahidmizbani** [ToshiSoftware/BB_PREAMP_PedalCtrl#1]
hello, i want to build your presetable BB pedal but cand find the LCD, can i use another LCD without change any program ? a regular 16*2 character LCD with I2C conection
thanks

**Comment by ToshiSoftware** [comment]
Yes you can use a general LCD device with I2C. If you make a PCB using the PCB data posted on the site below, please be careful as the shape of the LCD will not match.
http://toshi.life.coocan.jp/review/en_diy_arduino_bbpreamp.html

**Comment by vahidmizbani** [comment]
hello, i have an error when i want to compile the sketch, cant find the ST7032.h 
in arduino IDE just have 2 library for ST7032 and i installed all
but have error in compile

**Comment by vahidmizbani** [comment]
can u just send me all the libraries you used in this project ?

**Comment by vahidmizbani** [comment]
fatal error: ST7032.h: No such file or directory
 #include <ST7032.h>
          ^~~~~~~~~~
compilation terminated.

exit status 1

Compilation error: ST7032.h: No such file or directory

**Comment by vahidmizbani** [comment]
for LCD i cant find aqm1602 and just find general LCD device with I2C but with diffrent controller HD44780 controller can I use it without change your code ?
https://www.switchelectronics.co.uk/blue-16x2-i2c-lcd-display-module
